lesson6-questions.txt

```diff
@@ -3,36 +3,40 @@ Questions for Lesson 6
 1. When you enable CRUD operations on a table in the database, you need to add four things
 to your Rails application.  When you generate a scaffold, all four are created.What are they?
 
-
+model, controller, routes, and views
 
 
 2. When you specify a route in config/routes.rb, you need to include certain information.  What
 must be included? In addition, you may specify an additional parameter to create a variable
 that describes the route.  How do you do that? What variable is created?
 
-
+a verb, a path, and a to. the additional parameter is as.
 
 3. What does the Rails db:migrate command do?
 
-
+it changes the database based on the migrate file
 
 4. What file do you look at to see the current database schema?
 
+schema.rb
 
 5. What command shows the routes?
 
+bin/rails routes
 
 6. How can you start the Rails console? How can you do CRUD operations from the console?
 
+bin/rails server or bin/rails s. bin/rails console and then irb-like commands
```

Comment:
Heads up that the question just asks how to start Rails console and do CRUD from the console. So only the second part of your answer is relevant, since `bin/rails s(erver)` starts a Rails server.